### ISSUE TITLE
Bug in oiSequence.visualize

### ISSUE THREAD

**Issue by wandell** [isetbio/isetbio#172]
Line 61
In running s_vaLineOffset, although the oiSequence was generated the the time axis did not get set for  on the oiSequence.  We should check the oiSequence time axis as we create, or something.  This is just a note for one of us to track this down fix.

I noticed the error in s_vaLineOffset.m, when calling oiSequence.visualize;  So it could be an error in the vernier alignment code, or in the oiSequence code.  Hence, I linked us all into the assignment.

**Comment by wandell** [comment]
OK, sorry for the spamming.  Here's the deal.  HJ set the oiTimeAxis to be a single number instead of a vector.  This was not checked in the oiSequence create function.  So, @jamesgolden1 and I did the following:

   If a single number is set, we extend it to a vector equal to    

    (1:length(ModulationFunction))*thatSingleNumber

If it is a vector, we inserted a check that its length matches the modulationFuncton length
Otherwise, we throw an error.

We think some of the checking in the create should be handled by validation functions.  But we will do this later and it's not urgent.

t_oiSequence runs fine after these changes.

Closing.
